docs/docs/cmd/spo/web/web-get.md

```diff
@@ -32,6 +35,12 @@ Retrieve information about the site _https://contoso.sharepoint.com/subsite_ alo
 m365 spo web get --url https://contoso.sharepoint.com/subsite --withGroups
 ```
 
+Retrieve information about the site _https://contoso.sharepoint.com/subsite_ along with the RoleAssignments for the web
```

Comment:
also just noticed this now but along the way we may refactor and also in the new example use more general descriptions without duplicating the values which are in the code example